`assert_rust_compile` directly calls `rustc` without applying RUSTFLAGS.
Without taht, the --sysroot flag is missing when cross-compiling.
Just skip the check, we know the compiler works anyways.
--- a/build/moz.configure/rust.configure
+++ b/build/moz.configure/rust.configure
@@ -529,7 +529,6 @@
 @checking("for rust target triplet")
 def rust_target_triple(rustc, target, rust_supported_targets, arm_target):
     rustc_target = detect_rustc_target(target, arm_target, rust_supported_targets)
-    assert_rust_compile(target, rustc_target, rustc)
     return rustc_target
 
 
